src/Attributes/UnixTime.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+
+declare(strict_types=1);
+
+namespace Crell\Serde\Attributes;
+
+use Attribute;
+use Crell\AttributeUtils\SupportsScopes;
+use Crell\Serde\TypeField;
+
+#[Attribute(Attribute::TARGET_PROPERTY)]
+class UnixTime implements TypeField, SupportsScopes
```

Comment:
All of the other field types have a `Field` suffix on them.  Whether that was wise of me or not is debatable (mainly I wanted to avoid collisions if PHP ever adds Sequences and Dictionaries), we should probably be consistent in the naming.